README.md

```diff
@@ -3,45 +3,67 @@
 
-If you are already in the project folder, you can simply run `gocrane`.
+* `exclude-resource` - This flag specifies a folder, file, or glob pattern for files that should not be considered as resources, even if they match a `resource` flag value. It can be specified multiple times. By default GoCrane has this file set to a number of common files (e.g. `Dockerfile`, `README.md`, `.gitignore`) that are unlikely to be used by your application. If you set this flag, you would need to list your own defaults.
 
-The tool supports the following flags for more custom configuration:
+* `main` - This flag specifies the folder where your application's main package is located. Unlike previous flags, this one can point to a location that is not specified through a `dir` flag, however, this would rarely ever be meaningful, since it is likely that you would like to have GoCrane rebuild and restart your application when a Go file in the main package changes.
 
-* `--verbose, -v` - Enables verbose logging. Good for troubleshooting issues.
-* `--path, -p` - Specifies a path to watch recursively. This flag can be specified multiple times if multiple root directories should be watched.
-* `--exclude, -e` - Specifies a path to exclude from watching. This flag can be specified multiple times if multiple directory branches are to be excluded.
-* `--glob-exclude, -ge` - Specifies a glob pattern for files and directories that should be excluded. Can be specified multiple times. The pattern works only for a single path segment (e.g. `vendor`, `.DS_Store`, `*_test.go`)
-* `--run, -r` - Specifies the folder that includes the `main` package to be run.
-* `--cache, -c` - Specifies a pre-build executable to use the first time. This is useful when using `gocrane` with Docker images that have the built executable available to avoid unnecessary initial builds on startup.
-* `--args` - Specifies arguments to pass when starting the built executable.
-* `--build-args` - Specifies arguments to pass to `go build` (when compiling the executable)
+* `binary` - This flag specifies an executable that GoCrane should use when starting up, instead of rebuilding your application, as this could be a CPU-intensive application, especially if you have multiple GoCrane-managed applications starting at the same time. You should only specify this flag with the `gocrane run` command if the binary you reference has been built with `gocrane build`, since GoCrane would look for a `<executable>.dig` file to compare digest sums. If the digest sums don't match (which means that the source code you have mounted in the container has changed since `gocrane build` was used), GoCrane would default to triggering a rebuild and will not use the executable.
```

Comment:
```suggestion
* `binary` - This flag specifies an executable that GoCrane should use when starting up, instead of rebuilding your application, as this could be a CPU-intensive operation, especially if you have multiple GoCrane-managed applications starting at the same time. You should only specify this flag with the `gocrane run` command if the binary you reference has been built with `gocrane build`, since GoCrane would look for a `<executable>.dig` file to compare digest sums. If the digest sums don't match (which means that the source code you have mounted in the container has changed since `gocrane build` was used), GoCrane would default to triggering a rebuild and will not use the executable.
```